pathd/path_pcep_controller.c

```diff
@@ -191,6 +191,7 @@ int pcep_ctrl_initialize(struct thread_master *main_thread,
 	ctrl_state->pcc_count = 0;
 	ctrl_state->pcc_opts =
 		XCALLOC(MTYPE_PCEP, sizeof(*ctrl_state->pcc_opts));
+	memset(ctrl_state->pcc, 0, sizeof(ctrl_state->pcc[0]) * MAX_PCC);
```

Comment:
XCALLOC give you zeroed memory, the C means "cleared", so I don't think the memset is needed.

Reply:
that were for pcc array not for pcc_opts